gensim/models/base_any2vec.py

```diff
@@ -622,6 +718,7 @@ def __init__(self, sentences=None, input_streams=None, workers=3, vector_size=10
         self.running_training_loss = 0
         self.min_alpha_yet_reached = float(alpha)
         self.corpus_count = 0
+        self.corpus_total_words = 0
```

Comment:
Need to modify `load` function + add test (with loading old w2v model with new code, because you add new parameter)

Reply:
done